docs/Installing.asciidoc

```diff
@@ -20,74 +20,34 @@ has already read the <<GettingStarted.asciidoc#gettingstarted,Getting Started
 Guide>>, also available at the https://github.com/os-autoinst/openQA[official
 repository].
 
-Continue with the next section to get a simple, ready-to-use installation,
-useful for a single user setup. To setup using containers follow the
-<<#container_setup,Container based setup>>. Else, continue with the more
-advanced section about <<#custom_installation,Custom installation>>. For a
-setup suitable to develop openQA itself, have a look at the
+Continue with the next section <<#container_setup,Container based setup>> to
+setup a simple, ready-to-use container based openQA instance. useful for a
+single user setup. For a quick boostrapping under openSUSE go to
+<<#bootstrapping,Quick bootstrapping>>. Else, continue with the more advanced
+section about <<#custom_installation,Custom installation>>. For a setup
+suitable to develop openQA itself, have a look at the
 <<Contributing.asciidoc#development-setup,Development setup>> section.
 
-[[bootstrapping]]
-== Quick bootstrapping under openSUSE
-
-To quickly get a working openQA installation, you can use the openqa-bootstrap
-script. It essentially automates the steps mentioned in the
-<<#custom_installation,Custom installation>> section.
-
-=== Directly on your machine
-
-On openSUSE Leap and openSUSE Tumbleweed to setup openQA on your machine
-simply download and execute the openqa-bootstrap script as root - it will do
-the rest for you:
-
-[source,sh]
--------------------------------------------------------------------------------
-curl -s https://raw.githubusercontent.com/os-autoinst/openQA/master/script/openqa-bootstrap | bash -x
--------------------------------------------------------------------------------
-
-The script is also available from an openSUSE package to install from:
+[id="container_setup"]
+== Container based setup
 
-[source,sh]
--------------------------------------------------------------------------------
-zypper in openQA-bootstrap
-/usr/share/openqa/script/openqa-bootstrap
--------------------------------------------------------------------------------
+=== Single-instance container
```

Comment:
It is not very nice to start a sub-section immediately at the start of a section. Maybe move the sentence "openQA is provided in containers." up here?